htdocs/modules/plugins/templates/admin/plugins_admin_plugins.tpl

```diff
@@ -0,0 +1,36 @@
+<{include file="admin:system/admin_navigation.tpl"}>
```

Comment:
Not something to fix, just an FYI

In XoopsTpl we register a pre-filter with Smarty that converts the '<{' and '}>' delimiters to '{' and '}'. We use the standard Smarty default delimiters now in 2.6, but still support the old way for BC.

This change, combined with the switch to the .tpl extension) makes it a little easier to use existing Smarty documentation and examples, plus template aware editors (like PHPStorm) actually work much better.